utils/frame/remote-externalities/src/lib.rs

```diff
@@ -375,49 +376,62 @@ impl Builder {
 	}
 }
 
-#[cfg(feature = "remote-test")]
 #[cfg(test)]
-mod tests {
-	use super::*;
+mod test_prelude {
+	pub use super::*;
+	pub use sp_runtime::testing::{H256 as Hash, Block as RawBlock, ExtrinsicWrapper};
 
-	fn init_logger() {
+	pub type Block = RawBlock<ExtrinsicWrapper<Hash>>;
+
+	pub fn init_logger() {
 		let _ = env_logger::Builder::from_default_env()
 			.format_module_path(false)
 			.format_level(true)
 			.try_init();
 	}
+}
 
-	#[async_std::test]
-	async fn can_build_one_pallet() {
+#[cfg(test)]
+mod tests {
+	use super::test_prelude::*;
+
+	#[tokio::test]
+	async fn can_load_cache() {
 		init_logger();
-		Builder::new()
-			.mode(Mode::Online(OnlineConfig {
-				modules: vec!["Proxy".into()],
-				..Default::default()
+		Builder::<Block>::new()
+			.mode(Mode::Offline(OfflineConfig {
+				cache: CacheConfig { name: "test_data/proxy_test".into(), ..Default::default() },
 			}))
 			.build()
 			.await
 			.unwrap()
 			.execute_with(|| {});
 	}
+}
 
-	#[async_std::test]
-	async fn can_load_cache() {
+#[cfg(feature = "remote-test")]
+#[cfg(test)]
```

Comment:
Maybe:
```suggestion
#[cfg(all(test, feature = "remote-test"))]
```

…although it's a tossup which is more readable. :)